### PR TITLE
[Merged by Bors] - Improve docs around user auth

### PR DESCRIPTION
**Pull request by sbernauer**
## Description

*Please add a description here. This will become the commit message of the merge request later.*

## Review Checklist
- [x] Code contains useful comments
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/ROOT/pages/usage.adoc [hunk 1]
```diff
@@ -109,9 +109,44 @@ If the examples were loaded then some dashboards are already available:
 
 image::superset-dashboard.png[Superset dashboard showing birth names]
 
-== Authenticate users using LDAP
+== Authentication
+Every user has to authenticate itself before using Superset.
```
Comment (adwk67) [inline docs/modules/ROOT/pages/usage.adoc:113]:
```suggestion
Every user has to authenticate themselves before using Superset.
```

File: docs/modules/ROOT/pages/usage.adoc [hunk 2]
```diff
@@ -109,9 +109,44 @@ If the examples were loaded then some dashboards are already available:
...
+spec:
+  version: 1.4.1
+  [...]
+  authenticationConfig:
+    authenticationClass: superset-with-ldap-server-veri-tls-ldap
+    userRegistrationRole: Admin
+----
+
+== Authorization
+Superset has a concept called `Roles` which allows you to grant user permission based on roles.
```
Comment (adwk67) [inline docs/modules/ROOT/pages/usage.adoc:139]:
```suggestion
Superset has a concept called `Roles` which allows you to grant user permissions based on roles.
```

File: docs/modules/ROOT/pages/usage.adoc [hunk 3]
```diff
@@ -109,9 +109,44 @@ If the examples were loaded then some dashboards are already available:
...
+Have a look at the https://superset.apache.org/docs/security[Superset documentation on Security].
+
+=== Webinterface
+You can see all the available roles in the Webinterface of Superset.
+There you can also assign users to roles.
```
Comment (adwk67) [inline docs/modules/ROOT/pages/usage.adoc:144]:
```suggestion
You can view all the available roles in the Webinterface of Superset and can also assign users to these roles.
```

File: docs/modules/ROOT/pages/usage.adoc [hunk 4]
```diff
@@ -109,9 +109,44 @@ If the examples were loaded then some dashboards are already available:
...
+
+=== LDAP
+Superset supports assigning roles to users based on their LDAP group membership.
+This is not supported yet.
+Currently all the users logging in via LDAP all get assigned the same role which you can configure via the attribute `authenticationConfig.userRegistrationRole` on the `SupersetCluster` object.
```
Comment (adwk67) [inline docs/modules/ROOT/pages/usage.adoc:149]:
```suggestion
Currently all the users logging in via LDAP get assigned the same role which you can configure via the attribute `authenticationConfig.userRegistrationRole` on the `SupersetCluster` object.
```

**Review by adwk67 — APPROVED** [review]
Just made a few minor suggestions.

**Comment by sbernauer** [comment]
Thanks a lot!

**Pull request closed without merging**